File: Java Basics/01.Intro-Java-Homework/src/SumTwoNumbers.java
// Task 6:	Write a program SumTwoNumbers.java that enters two integers 
//			from the console, calculates and prints their sum. 

import java.util.Scanner;

public class SumTwoNumbers {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		System.out.print("Enter first number: ");
		int sum = scanner.nextInt();
		System.out.print("Enter second number: ");
		sum += scanner.nextInt();
		System.out.printf("The sum of both numbers is %s", sum);
	}
}

File: PHP Basics/01.PHP Syntax/02.Sum Two Numbers/SumTwoNumbers.php
<?php
// Task 02: Write a PHP script SumTwoNumbers.php that decleares two variables, firstNumber
//          and secondNumber. They should hold integer or floating-point numbers (hard-coded values).
//          Print their sum in the output in the format shown in the examples below. The numbers
//          should be rounded to the second number after the decimal point. Find in Internet how
//          to round a given number in PHP.

$firstNumber = 1.567808;
$secondNumber = 0.356;
echo '$firstNumber + $secondNumber = ';
echo "$firstNumber + $secondNumber = ";
echo number_format((double)($firstNumber + $secondNumber), 2, '.', '');
?>